packages/core/examples/video/index.ts

```diff
@@ -100,13 +101,29 @@ addButtonToToolbar({
   },
 });
 
+const useLocal = false;
+
 /**
  * Runs the demo
  */
 async function run() {
   // Init Cornerstone and related libraries
   await initDemo();
 
+  // Get Cornerstone imageIds and fetch metadata into RAM
+  const imageIds = await createImageIdsAndCacheMetaData({
+    StudyInstanceUID: '2.25.96975534054447904995905761963464388233',
+    SeriesInstanceUID: '2.25.15054212212536476297201250326674987992',
+    wadoRsRoot: useLocal
+      ? 'http://localhost:5000/dicomweb'
```

Comment:
use the other approach

Reply:
Will do as soon as it is available to be merged/used.